packages/nextjs/hooks/scaffold-eth/useNativeCurrencyPrice.ts

```diff
@@ -11,24 +11,23 @@ const enablePolling = false;
  */
 export const useNativeCurrencyPrice = () => {
   const { targetNetwork } = useTargetNetwork();
+  const [isFetching, setIsFetching] = useState(true);
   const [nativeCurrencyPrice, setNativeCurrencyPrice] = useState(0);
```

Comment:
Realised that this state are same as global state. 

`useNativeCurrencyPrice` is only used in `ScaffoldEthAppWithProvider`

Maybe rename this  to `useInitializeNativeCurrencyPrice` ? And instead of using `useState` maybe use states from store and set them here ? 

Doing this also clean up `ScaffoldEthAppWithProvider` because there we again don't need to set the global state , we just call this hook at top. 

What do you think ? M I missing something ?

Reply:
You're right, thanks! I thought about it but forgot/for some reason left it. Updated! [1460c4d](https://github.com/scaffold-eth/scaffold-eth-2/pull/859/commits/1460c4d68d7cc514d4c1ca9e3ee9b2c44eebc9a3)